File: src/server/Hoard.Core/Application/Portfolios/PortfolioDetailDto.cs
namespace Hoard.Core.Application.Portfolios;

public class PortfolioDetailDto
{
    public int Id { get; set; }
    public required string Name { get; set; }
    public bool IsActive { get; set; }
    public DateTime CreatedUtc { get; set; }
    
    public PortfolioPerformanceDto? Performance { get; set; }
}

File: src/client/hoard-ui/src/api/dtos/PortfolioDetailDto.ts
import type { PortfolioPerformanceDto } from "./PortfolioPerformanceDto";

export interface PortfolioDetailDto {
  id: number;
  name: string;
  isActive: boolean;
  createdUtc: string; // ISO string
  performance: PortfolioPerformanceDto | null;
}
